config/github/src/test/kotlin/GitHubConfigFileProviderTest.kt

```diff
@@ -47,7 +47,8 @@ import io.ktor.http.HttpStatusCode
 import io.mockk.every
 import io.mockk.mockk
 
-import java.util.Base64
+import kotlin.io.encoding.Base64
```

Comment:
What is the advantage from using Kotlin's Base64 encoder in code that not required to be multiplatform compatible?
Isn't it worse to use it while it is considered an `ExperimentalEncodingApi`?

Reply:
The above aside, the code is even a bit simpler now.

Reply:
@MarcelBochtler, can we move this forward, please?

Reply:
I'm not blocking this PR, but your arguments didn't convince me.
I would like to hear other opinions, e.g. from @mnonnenmacher or @oheger-bosch.

I will be fine with any decision.

Reply:
To me the risk is minimal, the worst that can happen is that we have to do some minor adaptations if the API changes with a Kotlin upgrade. And I would agree that it is in general a good practice to prefer Kotlin APIs over Java APIs when possible. Based on that I'll approve the change and merge later this afternoon if there are no objections.

Reply:
No objections from my side.